code/controllers/configuration.dm

```diff
@@ -519,6 +521,15 @@
 					config.silicon_max_law_amount	= text2num(value)
 				if("join_with_mutant_race")
 					config.mutant_races				= 1
+				if("roundstart_races")
+					if(!cleared_default_races)
+						roundstart_species = list()
+						cleared_default_races = 1
+					var/race_name = lowertext(value)
+					for(var/speciestype in typesof(/datum/species) - /datum/species)
+						var/datum/species/S = new speciestype()

```

Comment:
I DON'T KNOW HOW INITIAL() WORKS AND I'M VAGUELY CERTAIN IT MIGHT NOT WORK AT ALL GIVEN THE CIRCUMSTANCES